test/integration/include/find.test.js

```diff
@@ -172,6 +172,30 @@ describe(Support.getTestDialectTeaser('Include'), function() {
         });
     });
 
+    it('should include a model when the PK field name and foreign key name match', function() {
+      var A = this.sequelize.define('a', {
+        id: {
+          type: DataTypes.INTEGER,
+          field: 'a_id',
+          primaryKey: true
+        }
+      })
+      , B = this.sequelize.define('b', {});
+
+      A.hasOne(B);
+      B.belongsTo(A);
+
+      return this.sequelize
+        .sync({force: true})
+        .then(function() {
+          return A.create({ B: {} }, { include: [B] });
```

Comment:
How is this related to order and limit? Nested creation doesn't use any selects as far as I know